src/main/java/cz/cvut/kbss/termit/dto/RecentlyCommentedAsset.java

```diff
@@ -109,6 +109,7 @@ public String toString() {
         return "RecentlyCommentedAsset{" +
             "uri=" + uri +
             ", comment=" + lastComment.toString() +
+            ", myLastComment=" + myLastComment.toString() +
```

Comment:
Unnecessary calls to `toString` (inconsistent with the rest of the codebase and indeed the previous line as well).